spec/models/grda_warehouse/cas_active_spec.rb

```diff
@@ -69,12 +73,15 @@
     it 'finds one client who is active for CAS' do
       travel_to Time.local(2016, 2, 15) do
         expect(GrdaWarehouse::Hud::Client.destination.map(&:active_in_cas?).count(true)).to eq(1)
+        # Fails, cas_active scope does not respect ineligible_uses_extrapolated_days setting
+        # expect(GrdaWarehouse::Hud::Client.destination.cas_active.count).to eq(1)
```

Comment:
Not sure if this is a bug or on purpose

Reply:
I have a fix for this, but for some reason it's causing and issue with a different test and the transactions.  So forthcoming....